File: 1º Semestre/Programação de Computadores/Lista de Exercicios 1/13.cpp
#include <iostream>
#include <string>
using namespace std;

/*13.Escreva um programa que solicite ao usuário dois números inteiros e ao final apresente
na tela a soma dos dois números informados da seguinte forma: "O numeros N e X
somados correspondem a Y"*/

int main(){
	float numero1, numero2;
	
	printf("Digite o primeiro numero inteiro: ");
	scanf("%f", &numero1);
	
	printf("Digite o segundo numero inteiro: ");
	scanf("%f", &numero2);
	
	printf("Os numero %0.0f e %0.0f somados correspondem a %0.0f", numero1, numero2, numero1 + numero2);
}

File: 1º Semestre/Programação de Computadores/Lista de Exercicios 1/Python/13.py
"""
13.	Escreva um programa que solicite ao usuário dois números inteiros e ao final apresente na tela 
a soma dos dois números informados da seguinte forma: "O numeros N e X somados correspondem a Y" 
"""

intNum1 = int(input("Digite o primeiro numero inteiro: "))
intNum2 = int(input("Digite o segundo numero inteiro: "))

print("Os numeros {} e {} somados correspondem a {}".format(intNum1, intNum2, intNum1 + intNum2))